src/MongoDB/Query/Builder.php

```diff
@@ -145,6 +145,11 @@ public function groupBy($group)
         return $this;
     }
 
+    public function groupByWithCount($group)
```

Comment:
Method should return $this. And it should be added to Model.php PHPDoc according last changes